kafka-standalone/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# Linux for Health Kafka "stand-alone" iamge.
+#
+# This image provides a "stand-alone" Kafka implementation for non-production use.
+# Zookeeper is not bundled with this image. Zookeeper connection information is provided
+# via the KAFKA_ZOOKEEPER_CONNECT environment variable.
+#
+# Build arguments:
+# - KAFKA_RELEASE_URL: The URL for the target Kafka release
+#
+# Environment variables:
+# - APP_ROOT: The root application directory. Set in base image.
+# - USER_ID: The USER ID used for the non-privileged "application" account. Set in base image.
+# - JAVA_HOME: The Java installation directory. Set in base image.
+# - KAFKA_ZOOKEEPER_CONNECT: Specifies the Zookeeper host and port (maps to zookeeper.connect property)
+# - KAFKA_LISTENERS: Broker listening addresses (maps to listeners property)
+# - KAFKA_ADVERTISED_LISTENERS:  The Kafka listener metadata passed back to clients (maps to advertised.listeners)
+# - KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: Specifies the protocol used for each listener (maps to listener.security.protocol.map)
+# - KAFKA_INTER_BROKER_LISTENER_NAME:  The listener used for internal connections (inter.broker.listener.name)
+
+FROM docker.io/linuxforhealth/base:1.0.0  AS builder
+
+ARG KAFKA_RELEASE_URL=https://apache.osuosl.org/kafka/2.5.0/kafka_2.12-2.5.0.tgz
+
+RUN microdnf install -y wget tar gzip && \
+    microdnf clean all
+
+RUN mkdir -p /tmp/kafka
+WORKDIR /tmp
+RUN wget ${KAFKA_RELEASE_URL} -O /tmp/kafka.tgz && \
+    tar -xzf kafka.tgz -C kafka --strip-components 1
+COPY lfh-kafka-start.sh /tmp/kafka/bin
+
+FROM docker.io/linuxforhealth/openjdk:11
+
+LABEL maintainer="Linux for Health"
+LABEL com.linuxforhealth.component="kafka-standalone"
+LABEL name="kafka-standalone"
+LABEL com.linuxforhealth.license_terms="https://www.apache.org/licenses/LICENSE-2.0"
+LABEL summary="Kafka Standalone Deployment for Linux For Health"
+LABEL description="Provides a standalone deployment for non-production use"
+
+ENV APP_ROOT=${APP_ROOT}
+ENV USER_ID=${USER_ID}
+ENV JAVA_HOME=${JAVA_HOME}
+
+ENV KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+ENV KAFKA_LISTENERS=PLAINTEXT://:9092
+ENV KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://localhost:9092
+ENV KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=PLAINTEXT:PLAINTEXT
+ENV KAFKA_INTER_BROKER_LISTENER_NAME=PLAINTEXT
+
+RUN mkdir -p ${APP_ROOT}/kafka
+COPY --from=builder tmp/kafka ${APP_ROOT}/kafka
+RUN chmod u+x ${APP_ROOT}/kafka/bin/lfh-kafka-start.sh
+RUN chown lfh:root -R ${APP_ROOT}/kafka
+
+USER lfh
+
+# kafka broker 1
```

Comment:
expose is just used for documentation. Port mapping is done at runtime with
```
docker run -p <host port>:<container port>
```
We are just documenting that we may expose two ports, 9092 and 9094, based on the networking use-case.